.github/workflows/mysqldbdump.yml

```diff
@@ -1,8 +1,7 @@
 name: mysqldbdump
 on:
-  push:
-    branches:
```

Comment:
These PRs are now still getting made, but no one is approving and merging them -- is that intentional?

Recent PRs with this label: https://github.com/edx/edx-platform/pulls?q=label%3Amysqldbdump